README.md

```diff
@@ -1,92 +1,151 @@
+| `POSTGRESQL_VERSION` | `16` | `docker-compose.yml` | The postgresql version for the db container, the version should match the one required by the orchestrator version in use. |
+| `INMANTA_AUTHORIZED_KEYS` | / | `docker-compose.ssh.yml` | The public keys to insert into the ssh sidecar authorized keys for the inmanta user. |
+| `INMANTA_SSH_SIDECAR_IP` | `127.0.0.1` | `docker-compose.ssh.yml` | This environment variable specifies on which ip of the **host** the ssh sidecar should be made available. |
+| `INMANTA_SSH_SIDECAR_PORT` | `2222` | `docker-compose.ssh.yml` | This environment variable specifies on which port of the **host** the ssh sidecar should be made available. |
 
+## Composition
 
-## Deploy the oss orchestrator
+The files in this repo allow to deploy the orchestrator with different topologies.  The desired topology should be composed by passing more or less `docker-compose.*.yml` files to docker compose.  The file `docker-compose.yml` should always be provided.
 
+The docker compose commands can be summarized this way:
 ```bash
-# Latest oss release
-export INMANTA_ORCHESTRATOR_IMAGE=ghcr.io/inmanta/orchestrator:latest
-
-# Start db and orchestrator
-sudo docker compose up -d
-
-# Stop db and orchestrator
-sudo docker compose down
-
-# Clear storage or db and orchestrator
-sudo docker compose down -v
+sudo docker compose \
+    -f docker-compose.yml \
+    [-f docker-compose.iso.yml] \ # Deploy service orchestrator instead of oss one
+    [-f docker-compose.ssh.yml] \ # Deploy an ssh sidecar to access the orchestrator file system via ssh
+    [-f docker-compose.logrotate.yml] \ # Deploy a logrotate sidecar to rotate the logs of the orchestrator
+    <up|down|ps> [options...]
 ```
 
-## Deploy the oss orchestrator with an ssh sidecar
+## Examples
 
-:bulb: **To give access to your used in the ssh sidecar, you must provide some public key that will be installed in the container.  You can do this using the INMANTA_AUTHORIZED_KEYS environment variable.**
+### Deploy the oss orchestrator
 
 ```bash
 # Latest oss release
 export INMANTA_ORCHESTRATOR_IMAGE=ghcr.io/inmanta/orchestrator:latest
 
-# Your own, new-line separated, public key(s)
-export INMANTA_AUTHORIZED_KEYS="ssh-rsa ..."
+# Start db and orchestrator
+sudo docker compose -f docker-compose.yml up -d
 
-# Start db, orchestrator and ssh sidecar
-sudo docker compose -f docker-compose.yml -f docker.compose.ssh.override.yml up -d
+# Check the containers status
+sudo docker compose -f docker-compose.yml ps -a
 
-# Stop db, orchestrator and ssh sidecar
-sudo docker compose -f docker-compose.yml -f docker.compose.ssh.override.yml down
+# Stop db and orchestrator
+sudo docker compose -f docker-compose.yml down
 
 # Clear storage or db and orchestrator
-sudo docker compose -f docker-compose.yml -f docker.compose.ssh.override.yml down -v
+sudo docker compose -f docker-compose.yml down -v
```

Comment:
I wonder if we should make these individual code blocks to make clear they are not one sequence .

(and with that I mean, I really don't know what is best, just remarking it)

Reply:
I think I will keep it as it is for now, I like that it shows the shared context of the commands (the exported env vars are required for all commands, if we split them in different code blocks, we would need to provide them for every command block)